src/pages/alternatives.js

```diff
@@ -19,6 +24,11 @@ export const query = graphql`
         contact_phone
       }
     }
+    alternativesImage: imageSharp(id: {regex: "/alternatives1/"}) {
```

Comment:
Is there a way to specify the name exactly or is it standard with this library to use a regex to match the filename?